src/Screen.php

```diff
@@ -3,18 +3,54 @@
 namespace Native\Laravel;
 
 use Native\Laravel\Client\Client;
+use Illuminate\Support\Facades\Log;
 
 class Screen
 {
-    public function __construct(protected Client $client) {}
+  public function __construct(protected Client $client)
```

Comment:
Indentation seems to have changed from 4-space to 2-space throughout this file... can you change it back?